pennylane/devices/tests/test_compare_default_qubit.py

```diff
@@ -252,13 +252,13 @@ def test_four_qubit_random_circuit(self, device, tol):
         ]
 
         layers = 3
-        np.random.seed(1967)
-        gates_per_layers = [np.random.permutation(gates).numpy() for _ in range(layers)]
+        pnp.random.seed(1967)
+        gates_per_layers = [pnp.random.permutation(gates).numpy() for _ in range(layers)]
 
         def circuit():
             """4-qubit circuit with layers of randomly selected gates and random connections for
             multi-qubit gates."""
-            np.random.seed(1967)
+            pnp.random.seed(1967)
```

Comment:
Any worry here with global vs local RNGs?

Reply:
Updating to use a local generator for the permutation.